weed/mount/peer_registrar_test.go

```
@@ -0,0 +1,210 @@
+type fakeFilerClient struct {
+	filer_pb.SeaweedFilerClient // embed for methods we don't need
+
+	mu            sync.Mutex
+	registerCalls []filer_pb.MountRegisterRequest
+	listResponse  filer_pb.MountListResponse
+}
+
+func (f *fakeFilerClient) MountRegister(ctx context.Context, req *filer_pb.MountRegisterRequest, opts ...grpc.CallOption) (*filer_pb.MountRegisterResponse, error) {
+	f.mu.Lock()
+	defer f.mu.Unlock()
+	f.registerCalls = append(f.registerCalls, *req)
```

golangci-lint (2.11.4)

[error] copylocks: call of append copies lock value: github.com/seaweedfs/seaweedfs/weed/pb/filer_pb.MountRegisterRequest contains google.golang.org/protobuf/runtime/protoimpl.MessageState contains sync.Mutex

(govet)